test/node/animas/testAnimasSimulator.js

```diff
@@ -243,82 +243,58 @@ describe('animasSimulator.js', function() {
     });
 
     describe('status', function() {
-      var suspend = {
-        time: '2014-09-25T01:00:00.000Z',
-        deviceTime: '2014-09-25T01:00:00',
-        timezoneOffset: 0,
-        conversionOffset: 0,
-        deviceId: 'animas12345',
-        type: 'deviceEvent',
-        subType: 'status',
-        status: 'suspended',
-        reason: {suspended: 'automatic'}
-      };
-      var resume = builder.makeDeviceEventResume()
+      var suspend = builder.makeDeviceEventSuspend()
+        .with_time('2014-09-25T01:00:00.000Z')
+        .with_deviceTime('2014-09-25T01:00:00')
+        .with_timezoneOffset(0)
+        .with_conversionOffset(0)
+        .with_status('suspended')
+        .with_reason({suspended: 'manual'})
+        .done();
+      var suspendresume = builder.makeDeviceEventSuspendResume()
         .with_time('2014-09-25T02:00:00.000Z')
         .with_deviceTime('2014-09-25T02:00:00')
         .with_timezoneOffset(0)
         .with_conversionOffset(0)
         .with_status('resumed')
-        .with_reason({resumed: 'manual'});
+        .with_duration(3600000)
+        .with_reason({suspended: 'manual', resumed: 'manual'})
+        .done();
 
-      it('a suspend passes through', function() {
+      it('a single suspend passes through', function() {
```

Comment:
Shouldn't a single suspend not just pass through but have an annotation re: incomplete tuple? This is a comment on the test fixture I guess.